docs/scos/dev/example-modules.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Example modules
+description: Spryker example modules are designed for specific use cases and their functionality is aligned with unique business requirements.
+last_updated: Jan 11, 2024
+template: concept-topic-template
+---
+
+The Spryker *core modules* provide generic, reusable functionality that benefits a wide range of use cases. These modules offer a foundation for customization and serve as versatile building blocks.
+
+*Example modules* in contrast, are designed for specific, one-off use-cases. Their functionality is tightly aligned with unique business requirements and isn't suitable for broad reuse. These modules serve as specialized, project-specific solutions.
```

Comment:
@floriankraemer please add the example that you wanted to add instead of the Reach API

Reply:
Done.